go.mod

```diff
@@ -51,7 +51,7 @@ require (
 	code.cloudfoundry.org/bbs v0.0.0-20200403215808-d7bc971db0db
 	code.cloudfoundry.org/garden v0.0.0-20210208153517-580cadd489d2
 	code.cloudfoundry.org/lager v2.0.0+incompatible
-	github.com/DataDog/agent-payload/v5 v5.0.22
+	github.com/DataDog/agent-payload/v5 v5.0.23-0.20220607180631-ba597931aedc
```

Comment:
I'm going to update it once https://github.com/DataDog/agent-payload/pull/168 is merged.